jest.config.js

```diff
@@ -3,9 +3,7 @@
  * https://jestjs.io/docs/configuration
  */
 
-import type { Config } from '@jest/types';
-
-const config: Config.InitialOptions = {
+module.exports = {
```

Comment:
Yeah this is fine to convert this back. I don't think the types were giving us that much information anyway.